tests/foreman/longrun/test_oscap.py

```diff
@@ -6,34 +6,41 @@
 
 :CaseLevel: Acceptance
 
-:CaseComponent: UI
+:CaseComponent: CLI
 
 :TestType: Functional
 
-:CaseImportance: High
+:CaseImportance: Critical
 
 :Upstream: No
 """
+import os
```

Comment:
import required module only.
from os import path

Reply:
will look alternative to os , I think there already should be something in robottelo to get file path

Reply:
I disagree here. a strict requirement on only importing what modules are used isn't necessary. Additionally, it violates the rule of "explicit is better than implicit".
For example, it is more clear to do ```os.path(my_path_var)``` than ```path(my_path_var)```.
Though more people are familiar with the os library, this becomes especially true with libraries that aren't as common.

Reply:
@JacobCallahan Where do you stand with your opinion actually? First sentence says you disagree on it, you yourself explained in second sentence why you should agree and in last sentence, it seems you are agree with it :) and yes, more people are familiar with os module that's why the review comment is.

Reply:
@ntkathole I disagree with the requirement/block on merging because he just imported os. However, I'm clarifying my position that, if someone imports path from os in the future, I wouldn't block their PR on it.
Basically, in this instance, I'm ok with both ways and don't think that this should block a merge.

Reply:
@JacobCallahan @ntkathole I was looking for something in robottelo, I thought there already might be a helper written, but sadly there is none, but I think we can use split, as we already know the path of the file and not finding the filepath. Since we already know the filepath I will drop os.path library and use a single split statement to get file name.